internal/datastore/crdb/tx.go

```diff
@@ -184,9 +185,11 @@ func retriable(ctx context.Context, err error) bool {
 }
 
 func resetable(ctx context.Context, err error) bool {
-	// Ambiguous result error includes connection closed errors
-	// https://www.cockroachlabs.com/docs/stable/common-errors.html#result-is-ambiguous
-	return sqlErrorCode(ctx, err) == crdbAmbiguousErrorCode
+	// Unexpected EOF can be returned during high clock skew between nodes
+	return errors.Is(err, io.ErrUnexpectedEOF) ||
```

Comment:
Is there any way we can be slightly more specific on the error, even if it does mean string matching?

Reply:
Is this for high clock skew or for the connection being dropped?